tests/integration/fundMaliciousRedemption.test.js

```diff
@@ -1,9 +1,7 @@
 /*
  * @file Tests fund's ability to handle a malicious redemption attempts
  *
- * @test Redeem fails when malicious token is present
- * @test Redemption with non-registered assets does not succeed
- * @test redeemSharesWithConstraints succeeds to withdraw specific assets only
+ * @test Redeem succeeds when malicious token is present
```

Comment:
Let's organize this as 3 `test` blocks:

1. "Fund receives Malicious token" (you can keep as an investment for now, but this will need to change when only denomination asset investments are allowed. If you want to change this to a fund taking a trade, you can do that too.) -- fund receives token, then token starts reverting
2. "redeemShares fails"
3. "redeemSharesQuantity succeeds when _bypassFailure is set to true"